.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
all options in this file are redundant with `prettier` defaults but singleQuote.
>https://prettier.io/docs/en/options.html

Maybe you can remove those and only keep `singleQuote`

Reply:
Have to say I have no clue anymore, from which of my older projects I copied the prettierrc ; )...